low-level/framework/utils/utility.py

```diff
@@ -57,4 +61,71 @@ def is_env_vm(self):
         except Exception as e:
             logger.warning("Error while reading whether env is vm or not, assuming VM env : {e}")
         return is_vm
+    
+    @staticmethod
+    def send_command(command: str, fail_on_error=True):
+        # Note: This function uses subprocess to execute commands, scripts which are not possible to execute
+        # through any python routines available. So its usage MUST be limited and used only when no other
+        # alternative found.
+        output, error, returncode = SimpleProcess(command).run()
+        if returncode != 0:
+            print("command '%s' failed with error\n%s" % (command, error))
+            if fail_on_error:
+                sys.exit(1)
```

Comment:
There should be no print or exit from the middle of a library function. You should raise exception and caller will decide what to do with it.